exporter/dsn_fix/dsn_fix.go

```diff
@@ -36,7 +39,8 @@ func ClientOptionsForDSN(dsn string) (*options.ClientOptions, error) {
 	username := parsedDsn.User.Username()
 	password, _ := parsedDsn.User.Password()
 	if username != "" || password != "" {
```

Comment:
What if username is nil? It could be overwritten. 
Is it even possible? 
`if len(username) > 0 || leb(password) >0` {

Reply:
Resolved: It's string, not pointer to string.